ranger/config/rc.conf

```diff
@@ -401,6 +401,7 @@ map <F5> copy
 map <F6> cut
 map <F7> console mkdir%space
 map <F8> console delete
+map <F9> console trash
```

Comment:
If we're going to include an F-something key, it should me F8, since these keys are for "compatibility" with mc. We could comment it out. F9 just doesn't make any sense.

Reply:
But `<F8>`'s already taken. That's why I used the next in sequence.

Reply:
If we usurp `<F8>` we'll still have to use `<F9>` for `:delete` *and* we'll lose backwards compat.

Reply:
I'm not arguing against that but the reasoning sure should be solid.

Reply:
My reasoning is that the only reason we have the F5-F8 keys is the mc compatibility. It doesn't make sense to add a "non-standard" F9 key. If we'd comment out the alternative F8 binding, it won't clash with a regular deletion.